docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -552,3 +576,148 @@ const sleep = t => new Promise(res => setTimeout(res, t));
+      "type": "ios.app",
+      "build": "xcodebuild -workspace ios/eastestsexample.xcworkspace -scheme eastestsexample -configuration Debug -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build",
+      "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/eastestsexample.app"
+    }, /* @end */
+    "ios.release": {
+      "type": "ios.app",
+      "build": "xcodebuild -workspace ios/eastestsexample.xcworkspace -scheme eastestsexample -configuration Release -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build",
+      "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/eastestsexample.app"
+    },
+    /* @info New section */ "android.debug": {
+      "type": "android.apk",
+      "build": "cd android && ./gradlew :app:assembleDebug :app:assembleAndroidTest -DtestBuildType=debug && cd ..",
+      "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk"
+    }, /* @end */
+    "android.release": {
+      "type": "android.apk",
+      "build": "cd android && ./gradlew :app:assembleRelease :app:assembleAndroidTest -DtestBuildType=release && cd ..",
+      "binaryPath": "android/app/build/outputs/apk/release/app-release.apk"
+    }
+  },
+  "devices": {
+    "simulator": {
+      "type": "ios.simulator",
+      "device": {
+        "type": "iPhone 14"
+      }
+    },
+    "emulator": {
+      "type": "android.emulator",
+      "device": {
+        "avdName": "pixel_4"
+      }
+    }
+  },
+  "configurations": {
+    /* @info New section */ "ios.debug": {
+      "device": "simulator",
+      "app": "ios.debug"
+    }, /* @end */
+    "ios.release": {
+      "device": "simulator",
+      "app": "ios.release"
+    },
+    /* @info New section */ "android.debug": {
+      "device": "emulator",
+      "app": "android.debug"
+    }, /* @end */
+    "android.release": {
+      "device": "emulator",
+      "app": "android.release"
+    }
+  }
+}
+```
+</Collapsible>
+
+<Collapsible summary="eas-hooks/eas-build-on-success.sh">
+
+```sh
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+ANDROID_EMULATOR=pixel_4
+
+if [[ "$EAS_BUILD_PLATFORM" == "android" ]]; then
+  # Start emulator
+  $ANDROID_SDK_ROOT/emulator/emulator @$ANDROID_EMULATOR -no-audio -no-boot-anim -no-window -use-system-libs 2>&1 >/dev/null &
+
+  # Wait for emulator
+  max_retry=10
+  counter=0
+  until adb shell getprop sys.boot_completed; do
+    sleep 10
+    [[ counter -eq $max_retry ]] && echo "Failed to start the emulator!" && exit 1
+    counter=$((counter + 1))
+  done
+
+  if [[ "$EAS_BUILD_PROFILE" == "test" ]]; then
+    detox test --configuration android.release --headless
+  fi
+  ###### New section ######
+  if [[ "$EAS_BUILD_PROFILE" == "test_debug" ]]; then
+    detox test --configuration android.debug --headless
+  fi
+  #########################
+  # Kill emulator
+  adb emu kill
+else
+  if [[  "$EAS_BUILD_PROFILE" == "test" ]]; then
+    detox test --configuration ios.release --headless
+  fi
+  ###### New section ######
```

Comment:
@dsokal @amandeepmittal the `@info` annotation to highlight a new section doesn't seem to work in shell script text, so I used normal comments to highlight. Is there a better way to handle this?

Reply:
It should be working, you just need to stick to the hash for marking the comment:
* https://github.com/expo/expo/blob/main/docs/components/base/code.tsx#L152

If this syntax do not yield an expected result, we might have a regression from when we migrated to MDX v2.

Reply:
I see, will look at that soon :)

Reply:
Hmm, it looks like this syntax works here:
* https://docs.expo.dev/guides/config-plugins/#ios-cocoapods-podfile
* https://github.com/expo/expo/blob/8974afb9eb0dfb03cbfa79618d9cec57764c0148/docs/pages/guides/config-plugins.mdx?plain=1#L1013-L1015

Reply:
@douglowder, Simek has issued a fix. You can ship this, and I'll create a follow-up PR to add highlights for example comments.

Reply:
GH decided to close the PR because I have referenced comment from the review. 🤷‍♂️ Sorry about that.